internal/collector/collector.go

```diff
@@ -202,44 +202,46 @@ func (k *KeepalivedCollector) getKeepalivedStats() (*KeepalivedStats, error) {
 
 	var err error
 
+	stats.Scripts, err = k.collector.ScriptVrrps()
```

Comment:
Are we sure `keepalived.data` would be generated if the user asks the exporter to use the json signal?

P.S. I guess it's better to drop json support at all. It wouldn't bring any more metrics than the data signal...

Reply:
Not sure :) but the behavior hasn't changed. It was calling `ScriptVrrps` at the end before, and now It's gonna do it first. regardless of using JSON in both cases.

I'm not sure about the whole JSON support though. But let's leave that discussion to another issue :D

Reply:
No, unfortunately, we got a big `else` in that function (I remember you always tried to hesitate using `else` in codes :D) so it was under `!json`: https://github.com/mehdy/keepalived-exporter/blob/master/internal/collector/collector.go#L240

Reply:
F*ck! it was inside the `else` block =))
That's better, though! Let me change the code a little bit.

Reply:
Yeah :D - it was a part of this lint as well (https://github.com/mehdy/keepalived-exporter/blob/master/.golangci-lint.yml#L16) - we can probably remove it from the exclude if you could make it.